Dockerfile

```diff
@@ -29,9 +30,13 @@ RUN yumdownloader -x \*i686 --archlist=x86_64 \
     xz-libs \
     libprelude \
     gnutls \
-    nettle
+    nettle \
+    systemd-libs \
+    elfutils-libs \
+    lz4
 
 RUN rpm2cpio clamav-0*.rpm | cpio -vimd
```

Comment:
könnte man noch in ein run zusammenfassen

Reply:
Wollte ich auch machen, aber ich glaube man kann das Pipen nicht verketten, da das && sich dann auf den cpio command beziehen würde. Kann ich nochmal probieren